corehq/apps/app_manager/static/app_manager/js/report-module.js

```diff
@@ -89,7 +89,95 @@ var ReportModule = (function () {
         this.allGraphTypes = ['bar', 'time', 'xy'];
     }
 
-    function ReportConfig(report_id, display, availableReportIds, reportCharts, graph_configs, language) {
+    function FilterConfig(report_id, reportId, filterValues, reportFilters, changeSaveButton) {
```

Comment:
This function is long and deals with complex data...how about a comment explaining what the parameters are, so future devs can get a quick overview of what it's doing? This style can work well: https://github.com/dimagi/Vellum/blob/master/src/core.js#L1265-L1278